### PR TITLE
[circle-mpqsolver] Fix too long name

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit fixes too long name of the function.

See https://github.com/Samsung/ONE/pull/10432#discussion_r1109124953.

Draft: https://github.com/Samsung/ONE/pull/10432
Related: https://github.com/Samsung/ONE/issues/10393

ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: compiler/circle-mpqsolver/src/bisection/BisectionSolver.cpp [hunk 1]
```diff
@@ -30,8 +30,11 @@ using namespace mpqsolver::bisection;
 namespace
 {
 
-bool error_at_input_is_larger_than_at_output(const NodeDepthType &nodes_depth,
-                                             const std::string &visq_path, float cut_depth)
+/**
+ * @brief compare error of front half with an error of the rear part
+ */
+bool get_int16_at_front(const NodeDepthType &nodes_depth, const std::string &visq_path,
+                        float cut_depth)
```
Comment (jinevening) [inline compiler/circle-mpqsolver/src/bisection/BisectionSolver.cpp:37]: How about this?

```suggestion
/**
 * @brief Compare errors of two disjoint subsets of a model sliced by cut_depth
 * @return True if the front part (< cut_depth) has larger errors than the rear part (>= cut_depth)
 */
bool front_has_higher_error(const NodeDepthType &nodes_depth, const std::string &visq_path,
                        float cut_depth)
```

**Review by jinevening — APPROVED** [review]
LGTM

**Comment by jinevening** [comment]
@seanshpark is on leave today. I merge this without his approval as this change is quite simple.

**Review by seanshpark — COMMENTED** [review]
LGTM

**Pull request merged**